proxy/cc-proxy.spec-template

```diff
@@ -18,6 +18,7 @@ License  : Apache-2.0 GPL-2.0
 
 Requires: cc-proxy-bin
 Requires: cc-proxy-config
+Requires: clear-containers-selinux
```

Comment:
RedHat based-distros have selinux installed by defautl. Admins tend to disable it rather that uninstalling it. In any case, let me look if that kind of conditional exists.